test/git/Git.test.ts

```diff
@@ -16,11 +16,11 @@ const latestCommitHash = '547433c';
 const remote = 'git@github.com:tagoro9/fotingo-rewrite.git';
 
 const gitMocks = {
-  branch: jest.fn<Promise<unknown>, []>().mockResolvedValue({ all: ['remotes/origin/master'] }),
-  branchLocal: jest.fn<Promise<unknown>, []>().mockResolvedValue({ all: [] }),
-  checkoutBranch: jest.fn<Promise<unknown>, []>().mockResolvedValue(undefined),
-  fetch: jest.fn<Promise<unknown>, []>().mockResolvedValue(undefined),
-  getRemotes: jest.fn<Promise<unknown>, []>().mockResolvedValue([
+  branch: vi.fn().mockResolvedValue({ all: ['remotes/origin/master'] }),
```

Comment:
I think there might be some sort of typing difference with vi mocks vs jest mocks. I made this change to resolve type warnings.